.gitmodules

```
@@ -10,3 +10,6 @@
 [submodule "Externals/ICSharpCode.TextEditor"]
 	path = Externals/ICSharpCode.TextEditor
 	url = ../../gitextensions/ICSharpCode.TextEditor.git
+[submodule "Externals/EasyHook"]
+	path = Externals/EasyHook
+	url = git@github.com:gitextensions/EasyHook.git
```

```suggestion
	url = ../../gitextensions/EasyHook.git
```

and you need to commit the hash of the submodule... otherwise there is nothing to clone